doc/config_rsmtool.rst

```diff
@@ -25,106 +25,56 @@ test_file
-
-.. note::
+.. _feature_subset_file:
 
-    All responses with non-numeric values in either ``train_label_column`` or ``test_label_column`` and/or those with non-numeric values for relevant features will be automatically excluded from model training and evaluation. By default, zero scores in either ``train_label_column`` or ``test_label_column`` will also be excluded. See :ref:`exclude_zero_scores_rsmtool` if you want to keep responses with zero scores.
+feature_subset_file *(Optional)*
+""""""""""""""""""""""""""""""""
+Path to the feature subset file if using :ref:`subset-based column selection <subset_column_selection>`.
 
-.. _length_column_rsmtool:
+.. _feature_file_rsmtool:
 
-length_column *(Optional)*
-""""""""""""""""""""""""""
-The name for the optional column in the training and evaluation data containing response length. If specified, length is included in the inter-feature and partial correlation analyses. Note that this field *should not* be specified if you want to use the length column as an actual feature in the model. In the latter scenario, the length column will automatically be included in the analyses, like any other feature. If you specify ``length_column`` *and* include the same column name as  a feature in the :ref:`feature file <feature_file_rsmtool>`, ``rsmtool`` will ignore the ``length_column`` setting. In addition, if ``length_column`` has missing values or if its standard deviation is 0 (both somewhat unlikely scenarios), ``rsmtool`` will *not* include any length-based analyses in the report.
+features *(Optional)*
+"""""""""""""""""""""
+Path to the file with list of features if using :ref:`fine-grained column selection <feature_list_column_selection>`. Alternatively, you can pass a ``list`` of feature names to include in the experiment.
 
+.. _feature_fields_note:
 
-.. _second_human_score_column_rsmtool:
+.. note ::
 
-second_human_score_column *(Optional)*
-""""""""""""""""""""""""""""""""""""""
-The name for an optional column in the test data containing a second human score for each response. If specified, additional information about human-human agreement and degradation will be computed and included in the report. Note that this column must contain either numbers or be empty. Non-numeric values are *not* accepted. Note also that the :ref:`exclude_zero_scores_rsmtool` option below will apply to this column too.
+    By default, ``rsmtool`` will use all of the columns present in the training and evaluation files as features except for any columns explicitly identified in the configuration file (see below). The following four fields (``features``, ``feature_subset_file``, ``feature_subset``, and ``sign``) are useful if you want to use only a specific set of columns as features. See :ref:`selecting feature columns <column_selection_rsmtool>` for more details.
```

Comment:
I agree that the note should be moved. I am going to move it to right under `test_file` since the note refers to "training and evaluation files". I am also going to break up the note into two points, make the literals into links instead, and tweak the language a bit.